s5_base/local/make_samromur.sh

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
+# Copyright 2020 Reykjavik University (Judy Fong - judyfong@ru.is)
+# Apache 2.0.
+#
+# This script prepares the samromur dataset for kaldi for training, test, or
+# eval datasets. The new directory have utt2spk,spk2utt,text, wav.scp
+
```

Comment:
Just a small comment. make_samromur is not the most descriptive name. 

I would suggest something like samromur_data_prep.